Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al

```diff
@@ -301,6 +301,19 @@ codeunit 1990 "Guided Experience"
     begin
         GuidedExperienceImpl.AddTranslationForSetupObject(GuidedExperienceType, ObjectType, ObjectID, LanguageID, Translation, GuidedExperienceItem.FieldNo(Description));
     end;
+    
+    /// <summary>Adds the translation for the short title of the setup object.</summary>
+    /// <param name="GuidedExperienceType">The type of setup object.</param>/// 
```

Comment:
```suggestion
    /// <param name="GuidedExperienceType">The type of setup object.</param>
```

Reply:
a good catch from @darjoo

Reply:
oh right! 
I've copied it from the other procedure, and I've seen that there are other mistakes like this one

Reply:
haha that's true. didn't even notice them lol